src/styles/main.scss

```
@@ -1,3 +1,76 @@
+$size-cell: 100px;
+$gap: 1px;
+$col-count: 7;
+$width-field: $col-count * ($size-cell + $gap) - 1;
+$padding: 10px;
+
+$days:
+  "mon" 1,
+  "tue" 2,
+  "wed" 3,
+  "thu" 4, 
+  "fri" 5,
+  "sat" 6,
+  "sun" 7;
+
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $width-field + $padding * 2;
+  padding: $padding;
+
+  @each $day, $col in $days {
+    &--#{$day} :first-child {
+      margin-left: ($size-cell + $gap) * ($col - 1);
```

Начинай $days с 0, и тогда не надо будет делать -1
```suggestion
      margin-left: ($size-cell + $gap) * $col;
```